README.md

```diff
@@ -108,73 +116,58 @@ In these environments, two problems can be encountered, the solution for which i
+Tests, Examples and Benchmarks
+-----------------------------------
 
-### Ginkgo Testing
+### Testing
 
 Ginkgo does comprehensive unit tests using Google Tests. These tests are enabled by default and can be disabled if necessary by passing the `-DGINKGO_BUILD_TESTS=NO` to the cmake command. More details about running tests can be found in the [TESTING.md page](./TESTING.md).
 
-### Running the benchmarks
+### Running examples
+
+Various examples are available for you to understand and play with Ginkgo within the `examples/` directory. They can be compiled by passing the `-DGINKGO_BUILD_EXAMPLES=ON` to the cmake command. Documentation for the examples is available within the `doc/` folder in each of the example directory and a commented code with explanations can found in the [online documentation](https://ginkgo-project.github.io/ginkgo-generated-documentation/doc/develop/Examples.html).
+
+### Benchmarking
 
 A unique feature of Ginkgo is the ability to run benchmarks and view your results
 with the help of the [Ginkgo Performance Explorer (GPE)](https://ginkgo-project.github.io/gpe/).
 
 More details about this can be found in the [BENCHMARKING.md page](./BENCHMARKING.md)
 
-Contributing to Ginkgo
----------------------------
-
-### Contributing
-
-When contributing for the first time, please add yourself to the list of
-external contributors like in the example below.
-
-#### Contributors
-I hereby place all my contributions in this codebase under a BSD-3-Clause
-license, as specified in the repository's [LICENSE](./LICENSE) file.
-
-Name Surname <email@domain> Institution(s)
-
-#### Contributing guidelines
+Bug reports and Support
+-------------------------
 
-Contributing guidelines can be accessed in the [CONTRIBUTING.md
-page](./CONTRIBUTING.md). This page also contains other information useful to
-developers, such as writing proper commit messages, understanding Ginkgo's
-library design, relevant C++ information, and more.
+If you have any questions about using Ginkgo, please use [Github discussions](https://github.com/ginkgo-project/ginkgo/discussions).
 
-### Support
-If you have any question, bug to report or would like to propose a new feature,
-feel free to [create an
-issue on GitHub](https://github.com/ginkgo-project/ginkgo/issues/new). Another possibility
-is to send an email to [Ginkgo's main email address](mailto:ginkgo.library@gmail.com)
-or to contact any of the main [contributors](contributors.txt).
+If you would like to request a feature, or have encountered a bug, please [create an
+issue](https://github.com/ginkgo-project/ginkgo/issues/new). Please be sure to describe your problem and provide as much information as possible.
 
+You can also send an email to [Ginkgo's main email address](mailto:ginkgo.library@gmail.com) or to contact any of the main [contributors](contributors.txt).
```

Comment:
I'm not sure if advocating to email a contributor directly is a good choice. The list also contains previous contributors, who might not want to get these emails.